xla/mlir/tools/mlir_interpreter/dialects/BUILD

```diff
@@ -21,6 +22,7 @@ cc_library(
         ],
         exclude = ["util.cc"],
     ),
+    hdrs= ["symbol_finder.h"],
```

Comment:
This shouldn't be here. The `dialects` target should rather depend on `:find_symbol`